plover/gui/suggestions.py

```diff
@@ -161,7 +161,8 @@ def lookup_suggestions(self, phrase):
     def show_suggestions(self, suggestion_list):
 
         # Limit history.
-        if len(self.history) == HISTORY_SIZE:
+        undo_levels = self.config.get_undo_levels()
+        while len(self.history) >= undo_levels:
```

Comment:
This code does not work if undo levels is 0, fix [here](https://github.com/morinted/plover/compare/buffer...benoit-pierre:buffer).